src/Bicep.LangServer.IntegrationTests/CodeActionTests.cs

```diff
@@ -236,7 +236,7 @@ private async Task VerifyCodeActionIsAvailableToSuppressLinterDiagnostics(string
             var bicepConfigUri = DocumentUri.FromFileSystemPath(bicepConfigFilePath);
             fileSystemDict[bicepConfigUri.ToUri()] = bicepConfigFileContents;
 
-            var compilation = new Compilation(BicepTestConstants.Features, BicepTestConstants.NamespaceProvider, SourceFileGroupingFactory.CreateForFiles(fileSystemDict, uri, BicepTestConstants.FileResolver, BicepTestConstants.BuiltInConfiguration), BicepTestConstants.BuiltInConfiguration, BicepTestConstants.LinterAnalyzer);
```

Comment:
Can we add a test for the new code fix we are adding as part of this change?

Reply:
57f43c2a7 ("Fix fix title and add unit test", 2022-07-29)